en/datatypes/block.adoc

```diff
@@ -102,26 +102,34 @@ Not ok: `[42 pr   int "foo"]`
 
 == Accessing block elements
 
-Elements of block variables can be accessed using slash and index of an element:
+A block can be indexed with so-called _path notation_, using integer value for 1-based indexing:
```

Comment:
edit: pluralize: "integer values"

Do we need to day "so-called"? It sounds weak.